larq/quantizers.py

```diff
@@ -450,6 +452,90 @@ def __init__(
         )
 
 
+@utils.register_keras_custom_object
+class ReluSign(QuantizerFunctionWrapper):
+    r"""
+    Instantiates a binarization quantizer with output values 0 and 1.
+    \\[
+    q(x) = \begin{cases}
+    +1 & x > \Delta \\\
+    0 & x \leq \Delta
+    \end{cases}
+    \\]
+
+    where $\Delta$ is defined as the threshold and can be passed as an argument.
+    The gradient is estimated using the Straight-Through Estimator
+    (essentially the binarization is replaced by a clipped identity on the
+    backward pass).
+
+    \\[\frac{\partial q(x)}{\partial x} = \begin{cases}
+    1 & \left|x\right| \leq 1 \\\
+    0 & \left|x\right| > 1
+    \end{cases}\\]
+
+    ```plot-activation
+    quantizers.relu_sign
+    ```
+
+    # Arguments
+    threshold_value: The value for the threshold, $\Delta$, default value 0.
+    clip_value: Threshold for clipping gradients.
+
+    # Returns
+    AND Binarization function
+    """
+
+    def __init__(self, threshold_value=0.0, clip_value=1.0):
+        super().__init__(relu_sign, threshold_value=threshold_value, clip_value=clip_value)
+
+
+@utils.register_keras_custom_object
+@utils.set_precision(1)
+def relu_sign(x, threshold_value=0.0, clip_value=1.0):
```

Comment:
I am happy with going with ste_heaviside. ste_step and ste_unit_step I do not like since I would associate the sign function with it (especially unit_step seems just a bad name from the mathematical point of view since 0 is not a unit, I am surprised that people gave it this name).

Reply:
Sure, I'm fine with `ste_heaviside`